src/components/navigation/navigation-item/navigation-item.component.ts

```diff
@@ -0,0 +1,151 @@
+import {
+  Component,
+  Input,
+  ViewEncapsulation,
+  ContentChildren,
+  QueryList,
+  Renderer2,
+  ElementRef,
+  Optional,
+  SkipSelf,
+  AfterContentInit,
+  ContentChild,
+  Output,
+  AfterViewInit,
+  HostListener
+} from '@angular/core';
+import { Router, RouterLinkActive, NavigationEnd, RouterState, ActivatedRoute, UrlTree } from '@angular/router';
+import 'rxjs/add/operator/filter';
+
+@Component({
+    selector: '[ux-navigation-item]',
+    templateUrl: './navigation-item.component.html',
+    host: {
+        '[class.active]': 'active',
+        '[class.selected]': 'expanded',
+    },
+    encapsulation: ViewEncapsulation.None
+})
+export class NavigationItemComponent implements AfterViewInit {
+    @Input() header: string;
+    @Input() icon: string;
+    @Input() expanded: boolean = false;
+    @Input() link?: string;
+
+    @Input()
+    set level(value: number) {
+      this._level = value;
+    }
+
+    get level(): number {
+      return this._level;
+    }
+
+    @Output()
+    get active() {
+      if (this.link) {
+          return this._router.isActive(this.link, true);
+      } else {
+          return this._active;
+      }
+    }
+
+    private _level: number = 2;

```

Comment:
Why is this 2 by default?

Reply:
Default value of _level changed to 1. Along with a change to ngAfterViewInit() this will allow the assigment of classes like nav-second-level, nav-third-level, etc to match their distribution in the ng1 example.